app/views/layouts/application.html.erb

```diff
@@ -21,7 +65,9 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <a class="navbar-brand" href="/rooms">CRS2000</a>
+      <a class="navbar-brand" href="/rooms">
+        <img alt="Brand" src="<%= company_logo %>">
```

Comment:
What happens if we don't have a company logo uploaded or if we are not logged in? We should probably have some sort of fallback.